test/writeObfuscatedMockRac.ts

```diff
@@ -0,0 +1,36 @@
+import * as fs from 'fs';
+
+import { encodeBase64Hash, getMD5Hash } from '../src/obfuscation';
+
+import {
+  MOCK_RAC_RESPONSE_FILE,
+  OBFUSCATED_MOCK_RAC_RESPONSE_FILE,
+  TEST_DATA_DIR,
+  readMockRacResponse,
+} from './testHelpers';
+
+export function generateObfuscatedMockRac() {
+  const rac = readMockRacResponse(MOCK_RAC_RESPONSE_FILE);
+  const keys = Object.keys(rac.flags);
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const flagsCopy: Record<string, any> = {};
+  keys.forEach((key) => {
+    flagsCopy[getMD5Hash(key)] = rac.flags[key];
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    flagsCopy[getMD5Hash(key)].rules?.forEach((rule: any) =>
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      rule.conditions.forEach((condition: any) => {
+        condition['value'] = ['ONE_OF', 'NOT_ONE_OF'].includes(condition['operator'])
+          ? condition['value'].map((value: string) => getMD5Hash(value.toLowerCase()))
+          : encodeBase64Hash(`${condition['value']}`);
```

Comment:
```suggestion
          : encodeBase64Hash(condition['value']);
```

Reply:
Can't do this because encodeBase64 (`Buffer.from(input, 'utf8').toString('base64')`) only encodes strings and condition['value'] can be a number